index.js

```diff
@@ -1,169 +1,206 @@
-        break;
-      case 'speed':
-        vibration = data.speed / 100;
-        break;
-      case 'power':
-        vibration = Math.abs(data.power) / 1000000;
-        break;
-      case 'torque':
-        vibration = Math.abs(data.torque) / 1000;
-        break;
-      case 'brake':
-        vibration = data.brake / 255;
-        break;
-      case 'accel':
-        vibration = data.accel / 255;
-        break;
-    }
-
-    let vibrationInt = Math.round(vibration * maxVibration);
-    if (vibrationInt > maxVibration) vibrationInt = maxVibration;
-
-  }
-  sendVibration(vibrationInt);
-};
-
-(async function () {
-  console.log("Options:");
-  console.log(chalk.blue.bold("rpm            ") + "- RPM from the Engine " + chalk.bold("(Best all around experience)"));
-  console.log(chalk.blue.bold("rumble-average ") + "- Surface Rumble on Wheels " + chalk.italic("(Averaged out)"));
-  console.log(chalk.blue.bold("speed          ") + "- Speed, slow but fun");
-  console.log(chalk.blue.bold("power          ") + "- Not quite sure, seems to be a similar thing to rpm.");
-  console.log(chalk.blue.bold("torque         ") + "- Torque of the Vehicle");
-  console.log(chalk.blue.bold("brake          ") + "- How hard are you pushing down on the Brake Pedal");
-  console.log(chalk.blue.bold("accel          ") + "- How hard are you pushing down on the Accel Pedal");
-
-  var vibrationFrom = null;
-  while (vibrationFrom === null) {
-    console.log(chalk.bold("Select an option for Toys to React to?"))
-    vibrationFrom = prompt( chalk.green("> ") );
-
-    if (!vibrationFrom || acceptedValues.indexOf(vibrationFrom) === -1) {
-      vibrationFrom = null;
-      console.warn(
-        chalk.yellow("Invalid Value selected, please select a value from the list above")
-      );
-    }
-  }
-
-  var maxVibration = null; // Max Power of 1-20 (Use 20 if ya wanna go wild)
-  while (maxVibration === null) {
-    console.log(chalk.bold("What is the Max Vibration you would like to have? (1 to 20)"))
-    var stringMax = prompt( chalk.green("> ") );
-    maxVibration = Number(stringMax);
-
-    if (isNaN(maxVibration) || maxVibration < 1) {
-      maxVibration = null;
-      console.warn(
-        chalk.yellow("Please provide a Number between 1 & 20")
-      );
-    }
-    else if (maxVibration > 20) {
-      maxVibration = 20;
-      console.warn(
-        chalk.grey("Number exceeds 20, Setting value to 20")
-      );
-    }
-  }
-
-  await forza.loadGames();
-  forza.startAllGameSockets();
-  forza.on('telemetry', throttle(processData, 100));
-})();
+import chalk from 'chalk';
+import ForzaJS from 'forza.js';
+import nodeFetch from 'node-fetch';
+import promptSync from 'prompt-sync';
+
+const forza = new ForzaJS.default();
+const prompt = promptSync({ sigint: true });
+
+// TODO: swap to using Got instead of NodeFetch. it's much better
+// @ts-ignore
+const fetch = (...args) => nodeFetch.then(({ default: fetch }) => fetch(...args));
```

Comment:
This line seems to no longer function for me while testing on Version `v16.14.2`
 ```js
file:///G:/Documents/Github/lovense-forza-trappy/index.js:11
const fetch = (...args) => nodeFetch.then(({ default: fetch }) => fetch(...args));
                                     ^

TypeError: nodeFetch.then is not a function
    at fetch (file:///G:/Documents/Github/lovense-forza-trappy/index.js:11:38)
    at sendVibration (file:///G:/Documents/Github/lovense-forza-trappy/index.js:87:5)
    at Forza.processData (file:///G:/Documents/Github/lovense-forza-trappy/index.js:143:3)
    at Forza.<anonymous> (file:///G:/Documents/Github/lovense-forza-trappy/index.js:65:12)
    at Forza.emit (node:events:526:28)
    at Game.emit (node:events:538:35)
    at Game.onSocketMessage (G:\Documents\Github\lovense-forza-trappy\node_modules\forza.js\dist\Game.js:33:14)
    at Socket.emit (node:events:526:28)
    at UDP.onMessage [as onmessage] (node:dgram:922:8)
    at UDP.callbackTrampoline (node:internal/async_hooks:130:17)
```

This isn't a you blame this is just confusing me currently, as this worked fine before while testing.